app/src/main/java/com/g11x/checklistapp/Constants.java

```diff
@@ -0,0 +1,9 @@
+package com.g11x.checklistapp;
+
+/**
+ * Created by wcalabrese on 8/15/16.
+ */
+
+public class Constants {
+    public static final String LOCAL_REPOSITORY_AUTHORITY = "com.g11x.checklistapp.provider";
```

Comment:
In build.gradle add resValue("string", "content_provider_authority", "packagename")
That auto-adds it to the strings file and also lets you vary it based on build type. So you can install prod and non-prod versions at the same time (Content Provider authorities must be unique on a device).